### DISCUSSION TITLE
用命令行加入第二个账号以后浏览器无法访问文件

### DISCUSSION THREAD

**Discussion by clj83318** [messense/aliyundrive-webdav#597] [Q&A] [answered]
第一个账号正常能进去。第二个挂载和使用都没问题。就是浏览器X.X.X.X:端口进不去，大佬救救小白，能解决吗。还有怎么更改第二个账号的端口，小白直接默认的。把第一个账号的端口8080改了，笨办法。

**Comment by buuut** [comment, marked as answer]
自己摸索出来了，我是用群晖docker版本，
开了两个容器，容器里的端口设置需要注意下，本地端口可以随便改，但是容器端口只能是8080，不能修改。
这样看来，一个容器对应一个账号，解决了多账号的问题。
